lib/koala/api.rb

```diff
@@ -61,7 +61,7 @@ def api(path, args = {}, verb = "get", options = {}, &error_checking_block)
         end
 
         # Translate any arrays in the params into comma-separated strings
-        args = sanitize_request_parameters(args)
+        args = sanitize_request_parameters(args) unless Koala.configuration.allow_array_parameters
```

Comment:
For consistency, this should probably also be override-able on a per request basis. As you point out, specific APIs need this while others don't, so it's possible people might want the default behavior most of the time with specific overrides.